messages/autoware_lanelet2_msgs/msg/MapBin.msg

```diff
@@ -11,3 +11,6 @@ string map_version
 
 # binary data of lanelet2 map. This is meant to be filled using toBinMsg() in lanelet2_extension library
 int8[] data
+
+# route flag to set the route invalidation status
+bool route_flag
```

Comment:
lets call this invalidates_route so that the reason for it is a bit clearer. 
Can you also add the route_id and route_version (see cav_msgs/Route) to this message. I don't think we will need to use them directly yet, but it will be helpful for debugging to know which route was invalidated.